### PR TITLE
Refactor Code

### PR DESCRIPTION
**Pull request by Jerry-py**
The code is refactored but you might need to check if it works.

## Summary

This pull request is to refactor the code to be more readable and more simpler. The code hasn't been checked I don't know if it works or not.

Refactors:
- Fixed nested loops
- Use f-strings instead
- Use contextlib to suppress errors
- Merged list append
- Remove-unnecessary-else
- Remove redundant if
- Etc

- [x] If code changes were made then they have been tested.
- [ ] This PR fixes an issue.

### CODE REVIEW

**Review by WhoIsConch — COMMENTED** [review]
Looks pretty good, but please test the changes before we implement them.

**Comment by Jerrydotpy** [comment]
The bot inverted my commit. Do I have to redo everything? @WhoIsConch

**Comment by iayushanand** [comment]
> The bot inverted my commit. Do I have to redo everything? @WhoIsConch

it probably happened because a new pr was merged, I guess you gotta redo it. You can still look at your commit to make it easy

**Comment by Jerry-py** [comment]
@iayushanand It should be fixed and it works perfectly. Also the images are also optimized

**Comment by Jerry-py** [comment]
Don't merge yet because I am still improving the `README`

**Comment by Jerry-py** [comment]
@iayushanand do i still need a cooldown section to the commands table or its good enough

File: README.md [hunk 1]
```diff
@@ -1,2 +1,111 @@
...
+| None | Fun | **reverse** | Reverse inputted string | `{prefix}reverse [string]`: *returns string reversed* | 
+| None | Fun | **owofy** | Owofy inputted string | `{prefix}owofy [text]`: *returns owofy text* |
+| None | Fun | **mock** | Mockify inputted string | `{prefix}mock [text]`: *returns mocked text* |
+| None | Fun | **beerparty** | Start a beerparty 🍻!! | `{prefix}beerparty [reason]`: *start a beer party* |
+| None | General | **source** (github, code) | Get the source of this bot | `{prefix}source` *will send link to my source code* <br > `{prefix}source [command]` *will send link to the source code of the command* <br> `{prefix}source [command] [subcommand]`: *will send link to the source code of the subcommand* | 
+| None | General | **define** | Gets deinitions from Urban Dictionary. | `{prefix}define [word]`: *will send the definition of the word* |
+| None | General | **avatar** | Commands for getting avatars. | `{prefix}avatar`: *will send a list of available methods* |
+| None | General | **avatar main** | Start a beerparty 🍻!! | `{prefix}avatar main [user]`: *will send the main avatar of the user* |
```
Comment (iayushanand) [inline README.md]: fix the typo here:
image

**Review by iayushanand — COMMENTED** [review]
everything else lgtm, fix that and I'll merge

**Review by iayushanand — APPROVED** [review]
LGTM!

**Review by andrewthederp — APPROVED** [review]

**Pull request merged**